core/src/banking_stage/forwarder.rs

```diff
@@ -22,28 +22,47 @@ use {
     },
 };
 
-pub struct Forwarder;
+pub struct Forwarder {
```

Comment:
seems `pub(crate)` is enough? (could be ditto for `fn`s)

i know this is tedious but this hint is useful to scope down my limited mind capacity. ;)

Reply:
Done: https://github.com/solana-labs/solana/pull/29403/commits/130202357d2cc4b35742b443dacedb9d6d9d2c59

Reply:
^hey my commit hash is today's date! 🎉